src/webhooks/utils.tsx

```diff
@@ -1,20 +1,29 @@
 import { MultiAutocompleteChoiceType } from "@saleor/components/MultiAutocompleteSelectField";
+import PreviewPill from "@saleor/components/PreviewPill";
 import {
   WebhookEventTypeAsyncEnum,
   WebhookEventTypeSyncEnum,
   WebhookFragment,
 } from "@saleor/graphql";
+import React from "react";
 
 export function isUnnamed(webhook: WebhookFragment): boolean {
   return ["", null].includes(webhook?.name);
 }
 
+const isWebhookInPreview = (
+  webhook: WebhookEventTypeSyncEnum | WebhookEventTypeAsyncEnum,
+) =>
+  webhook === WebhookEventTypeSyncEnum.CHECKOUT_CALCULATE_TAXES ||
+  webhook === WebhookEventTypeSyncEnum.ORDER_CALCULATE_TAXES;
```

Comment:
A bit more scalable  ;)

```suggestion
[WebhookEventTypeSyncEnum.CHECKOUT_CALCULATE_TAXES, WebhookEventTypeSyncEnum.ORDER_CALCULATE_TAXES].includes(webhook);
```